modules/core/include/opencv2/core/utils/trace.hpp

```diff
@@ -106,7 +106,7 @@ enum RegionLocationFlag {
 
     REGION_FLAG_IMPL_IPP = (1 << 16),            ///< region is part of IPP code path
     REGION_FLAG_IMPL_OPENCL = (2 << 16),         ///< region is part of OpenCL code path
-    REGION_FLAG_IMPL_OPENVX = (3 << 16),         ///< region is part of OpenVX code path
+///    REGION_FLAG_IMPL_OPENVX = (3 << 16),         ///< region is part of OpenVX code path
```

Comment:
@mshabunin, @opencv-alalek, shall we keep this value for compatibility?

Reply:
I think we can remove it. Since it is 5.x we can break compatibility.

Reply:
@invarrow, please remove the line completely

Reply:
done